tests/release/release.go

```diff
@@ -128,7 +128,7 @@ var _ = framework.ReleaseSuiteDescribe("[HACBS-1108]test-release-service-happy-p
 				}
 
 				return prList.Items[0].HasStarted() && prList.Items[0].IsDone() && prList.Items[0].Status.GetCondition(apis.ConditionSucceeded).IsTrue()
-			}, avgPipelineCompletionTime, defaultInterval).Should(BeTrue())
+			}, releasePipelineRunCompletionTimeout, defaultInterval).Should(BeTrue())
 		})
 
 		It("The Release should have succeeded.", func() {
```

Comment:
same.

Reply:
Same comment as above I need to be specific